ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -334,7 +334,7 @@ SHARED_DRIVE_ROOT = '{{ shared_mount_dir }}'
 RESTORE_PAYLOAD_DIR_NAME = '{{ restore_payload_dir_name }}'
 TRANSFER_FILE_DIR_NAME = '{{ transfer_payload_dir_name }}'
 SHARED_TEMP_DIR_NAME = '{{ shared_temp_dir_name }}'
-TRANSFER_SERVER = '{{ localsettings.TRANSFER_SERVER }}'
+TRANSFER_SERVER = '{{ (proxy_type == 'nginx') | ternary('nginx', 'apache') }}'
```

Comment:
Previously there were two settings that needed to be changed 'proxy_type' and 'localsettings.TRANSFER_SERVER'. they both need to change when the proxy changes, otherwide async downloads wont work, but TRANSFER_SERVER expects 'apache' and proxy_type expects 'apache2'